app/src/pages/inside/common/integrations/availableIntegrations/availableIntegrations.scss

```
@@ -22,23 +22,52 @@
+  gap: 16px;
+  max-width: 820px;
   width: 100%;
   box-sizing: border-box;
-  padding: 0 32px;
-  margin: 24px auto 48px;
 }
 
 .integrations-header {
-  @include measure(15px, 24px, $COLOR--almost-black, $FONT-ROBOTO-MEDIUM);
+  @include measure(15px, 24px, var(--rp-ui-base-almost-black), var(--rp-ui-base-font-family));
+  font-weight: var(--rp-ui-base-fw-medium);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)